File: connectors/connector-github/src/test/scala/com/taskadapter/connector/github/MockUserService.scala
package com.taskadapter.connector.github

import org.eclipse.egit.github.core.User
import org.eclipse.egit.github.core.service.UserService

class MockUserService extends UserService {
  override def getUser(login: String): User = {
    new User()
      .setLogin(login)
      .setName("name")
  }
}

File: connectors/connector-github/src/test/java/com/taskadapter/connector/github/MockUserService.java
package com.taskadapter.connector.github;

import org.eclipse.egit.github.core.User;
import org.eclipse.egit.github.core.service.UserService;

public class MockUserService extends UserService {
    @Override
    public User getUser(String login) {
        return new User()
                .setLogin(login)
                .setName("name");
    }
}
